incubator/hnc/test/e2e/delete_crd_test.go

```diff
@@ -20,11 +20,6 @@ var _ = Describe("When deleting CRDs", func() {
 	})
 
 	AfterEach(func() {
-		// HNC might be in an arbitrarily bad state - e.g. the CRDs are gone so the reconcilers aren't
-		// running anymore, and the admission webhooks are operating off of bad information (e.g. that a
-		// subnamespace's annotation has been removed). So delete the VWC so we can clean up
-		// effectively; it'll be restored by RecoverHNC anyway.
-		MustRun("kubectl delete validatingwebhookconfigurations hnc-validating-webhook-configuration")
```

Comment:
No, we still need this here! Because if the test fails, we won't get to the end of the test to clean up properly. All cleanup *must* be in AfterEach.

But we should check to see if HNC_REPAIR is set and skip this line if it's not.

Reply:
Got it! Done